dds/DCPS/RemoveAssociationSweeper.h

```diff
@@ -36,6 +36,7 @@ class RemoveAssociationSweeper : public ReactorInterceptor {
 
   void schedule_timer(OpenDDS::DCPS::RcHandle<OpenDDS::DCPS::WriterInfo>& info, bool callback);
   ReactorInterceptor::CommandPtr cancel_timer(OpenDDS::DCPS::RcHandle<OpenDDS::DCPS::WriterInfo>& info);
+  void cancel_timer(OpenDDS::DCPS::PublicationId writer_id);
```

Comment:
I know you're following the pattern that's already here, but I don't know why the code in this file keeps obfuscating the interesting parts by repeating the full namespace name on every line (sometimes twice).